pkg/metrics/collector/pod_ip_metrics.go

```diff
@@ -124,6 +149,20 @@ type clock interface {
 
 type realClock struct{}
 
+type IPAMRange struct {
```

Comment:
Should we reuse (import) the types and logic from https://github.com/containernetworking/plugins/blob/main/plugins/ipam/host-local/backend/allocator/config.go instead of reinventing the wheel?

Reply:
Sure. I will reuse that. Sorry I can no longer modify this pull request. I opened a new pull request to change the logic here.